action.yml

```
@@ -160,8 +174,13 @@ runs:
           echo "Attempting to merge-up $FROM_BRANCH into $INTO_BRANCH"
 
           # Checkout both branches to ensure branch info is up to date
+          # Also git pull which should not be uneccessary though in practice GitHub actions sometimes throws
```

```suggestion
          # Also git pull which should not be necessary though in practice GitHub actions sometimes
```
"should not be unnecessary".... do you mean should not be necessary? Or should be necessary?

Also if the "throws" should be there then it's not clear to me what you mean by "throws rejects"

---

Updated